test-kysect/CommandsHandler.cs

```diff
@@ -52,33 +45,26 @@ public void Completed()
         
         public void Today()
         {
-            if (_tasks.Exists(t => t.Time.Day == DateTime.Now.Day && 
-                                 t.Time.Month == DateTime.Now.Month && 
-                                 t.Time.Year == DateTime.Now.Year &&
-                                 !t.Completed))
+            var todayTasks= _tasks.Where(t => t.Time.Day == DateTime.Now.Day &&
```

Comment:
```suggestion
            var todayTasks = _tasks.Where(t => t.Time.Day == DateTime.Now.Day &&
```
Не хватило пространства немного